java/ql/src/experimental/Security/CWE/CWE-470/LoadClassNoSignatureCheck.ql

```diff
@@ -70,16 +70,16 @@ predicate isSignaturesChecked(MethodAccess maCreatePackageContext)
 }
  
 from
-  MethodCall maCreatePackageContext, LocalVariableDeclExpr lvdePackageContext,
-  Expr sinkPackageContext, MethodCall maGetMethod, MethodCall maInvoke
+  MethodAccess maCreatePackageContext, LocalVariableDeclExpr lvdePackageContext,
+  DataFlow::Node sinkPackageContext, MethodAccess maGetMethod, MethodAccess maInvoke
```

Comment:
Changed `Expr sinkPackageContext` to `DataFlow::Node sinkPackageContext`, due to the following error:
`Error: Failed to run query: ERROR: Expr::Expr is not compatible with DataFlowNodes::Public::Node`